packages/server/__tests__/responder/component.test.ts

```diff
@@ -40,9 +41,9 @@ const newIdentifierAndTransaction = (blockNumber: number, data: string, from: st
         to,
         value
     };
-    const identifier = new PisaTransactionIdentifier(chainId, data, to, value, gasLimit);
+    const identifier = new PisaTransactionIdentifier(chainId, data, to, new BigNumber(value), new BigNumber(gasLimit));
```

Comment:
Check all usages of value and gaslimit, since we'll no longer need to do new BigNumber - even though that's valid and will compile.

Reply:
That was actually a BigNumber before, and I changed it to string in the Block data type (I "stringify" them in the block factories).